spec/models/user_spec.rb

```diff
@@ -6,4 +6,19 @@
   it { should validate_presence_of(:full_name) }
   it { should validate_uniqueness_of(:email) }
   it { should have_many(:queue_items).order(:position) }
+
+  describe "#queued_video?" do
+    it "should return if the item is in the queue" do
```

Comment:
I would recommend getting rid the "should" word from message. It is the old way of doing it, but when you think about it - it does not bring any value. The style now is using 3rd person of the verb right away - `it "returns ..." do`. Also Read more about it [here](http://www.betterspecs.org/#should).